### PR TITLE
Update new-db.md

### LINKED ISSUE #1474: Tutorial feels incomplete since no handling of Posts
The tutorial feels incomplete since there is no handling of Posts. This would be beneficial to show how relational data is handled. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c78ab24c-ddc8-521d-f032-68b9dfaab803
* Version Independent ID: 23d99a0f-1eb8-3f0b-5a09-f2520aa3b496
* Content: [Getting Started on ASP.NET Core - New database - EF Core](https://docs.microsoft.com/en-us/ef/core/get-started/aspnetcore/new-db?tabs=netcore-cli#feedback)
* Content Source: [entity-framework/core/get-started/aspnetcore/new-db.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/get-started/aspnetcore/new-db.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #1474

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  please review
the files changes is not correct because I did an accidental commit to master in my last edit. The MVC links at the bottom are new. I changed the order to so you could see the new links in the changed files.

File: entity-framework/core/get-started/aspnetcore/new-db.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ dotnet run
 
 ## Additional Resources
 
+* [Get started with ASP.NET Core MVC
+](/aspnet/core/tutorials/first-mvc-app/start-mvc) or [Get started with Razor Pages in ASP.NET Core](/aspnet/core/tutorials/razor-pages/razor-pages-start)
```
Comment (tdykstra) [inline entity-framework/core/get-started/aspnetcore/new-db.md]: The link got split up into two lines

**Comment by Rick-Anderson** [comment]
>  Does it make sense to have the same list of additional resources in other pages as well?

If you like, I could put that list in an include we could add to other tutorials. Let me know.

**Comment by divega** [comment]
> > Does it make sense to have the same list of additional resources in other pages as well?
> 
> If you like, I could put that list in an include we could add to other tutorials. Let me know.

Maybe, but I am not sure it is the only approach. I have created a separate issue for this:
https://github.com/aspnet/EntityFramework.Docs/issues/1499

Thanks for the PR!

**Pull request merged**